helix-core/src/test/java/org/apache/helix/manager/zk/TestZkBaseDataAccessor.java

```diff
@@ -433,6 +433,37 @@ public void testSyncRemove() {
     System.out.println("END " + testName + " at " + new Date(System.currentTimeMillis()));
   }
 
+  @Test
+  public void testDeleteLogging() {
+    String root = _rootPath;
+    _gZkClient.deleteRecursively("/" + root);
```

Comment:
I might have missed it last time. Why we are deleting everything under root?

Reply:
Also please link the issue number if the description.

Reply:
This is to make sure we work with a clean path. However, it might not be necessary now that you point it out.